crates/rustc_codegen_spirv/src/symbols.rs

```diff
@@ -118,8 +118,8 @@ const BUILTINS: &[(&str, BuiltIn)] = {
         ("layer_per_view_nv", LayerPerViewNV),
         ("mesh_view_count_nv", MeshViewCountNV),
         ("mesh_view_indices_nv", MeshViewIndicesNV),
-        ("bary_coord_nv", BaryCoordNV),
-        ("bary_coord_no_persp_nv", BaryCoordNoPerspNV),
+        ("bary_coord", BaryCoordKHR),
+        ("bary_coord_no_persp", BaryCoordNoPerspKHR),
```

Comment:
I've just removed the `_nv` suffix, which may break some code. Should we instead still allow the suffix and just replace it with the KHR version internally?

Reply:
NV aliases are still there in `rspirv::BuiltIn`, no change was needed here: <https://docs.rs/spirv/0.3.0+sdk-1.3.268.0/spirv/enum.BuiltIn.html#associatedconstant.BaryCoordNV>.

Well except this abuses glob imports, hah. The `use BuiltIn::*;` must've been why it started erroring.
But that's not new! You can see that some are referenced `BuiltIn::` without relying on the glob.